Makefile.am

```diff
@@ -98,6 +126,9 @@ EOS_COVERAGE_BLACKLIST_PATTERNS =	\
 AM_CFLAGS += @EOS_C_COVERAGE_CFLAGS@
 AM_LDFLAGS += @EOS_C_COVERAGE_LDFLAGS@
 
+# # # COMMON LIBRARY # # #
+include $(top_srcdir)/shared/Makefile.am.inc
```

Comment:
```
automake: error: cannot open < shared/Makefile.am.inc: No such file or directory
autoreconf: automake failed with exit status: 1
```

Reply:
Ouch, thanks for spotting - I rebased and put it into the right commit.